core/include/mm/mobj_ffa.h

```diff
@@ -0,0 +1,75 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
Please create this file in a separate commit so it's easier to see what's added specifically for SPs.